cdist/configuration.py

```diff
@@ -382,7 +382,11 @@ def __init__(self, command_line_args, env=os.environ,
             self.env = {}
         else:
             self.env = env
-        self.config_files = config_files
+        # skonfig override
+        if os.path.basename(sys.argv[0]) == "skonfig":
```

Comment:
To me this check looks a bit fragile. If someone makes a symlink which is not called `skonfig` the check will fall back to cdist mode.

Wouldn't it be better to set some global variable when `bin/skonfig` is launched and use its value for this check (e.g. `globals().get("cli", "cdist") == "skonfig"`)?